src/components/Comments.jsx

```diff
@@ -1,36 +1,86 @@
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { getComments } from "../utils/api";
+import { postComment } from "../utils/api";
+import { deleteComment } from "../utils/api";
 
 export default function Comments() {
   const [comments, setComments] = useState([]);
+
   const { article_id } = useParams();
 
   useEffect(() => {
-    getComments(article_id).then((body) => {
-      setComments(body);
-    });
+    getComments(article_id).then((body) => setComments(body));
   }, []);
 
+  function handleOpenAddCommentDlg() {
+    const addCommentDlg = document.getElementById("addCommentDlg");
+    addCommentDlg.showModal();
+  }
+
+  function handleCloseAddCommentDlg() {
+    const addCommentDlg = document.getElementById("addCommentDlg");
+    comment.value = "";
```

Comment:
We should not be doing the dom manipulation stuff on this project. 

For this task we need a state which is keeping track of what is being typed in our text area (with an onChange) 

Then when the form is submitted or button clicked an API request is made with this state to post the comment as you have done below.